includes/class-wpcom-legacy-redirector-cli.php

```diff
@@ -257,6 +257,8 @@ function import_from_csv( $args, $assoc_args ) {
 			WP_CLI::line( 'Processing...' );
 		}
 
+		ini_set('auto_detect_line_endings',TRUE);
```

Comment:
This is minor, it would be nice to space code per the WordPress PHP Coding Standards: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage.

Also, should we check that it isn't `TRUE` already before setting it and if so, should we set it back to its original value after processing?

Reply:
@rebeccahum Good catch, I was coding with Sublime Text instead of PHP Storm to save some resources. I've applied the code formatting. 👍 

Since this is an internal CLI function limited in scope to importing this CSV I don't believe its necessary. Also, the PHP documentation does not mention that it will be breaking change. What do you think?

Reply:
@bdtech Correct, it is an internal CLI function but it is also used on local test environments and leaving it on could interfere with other script executions.  Additionally, do you think there could be any performance implications with leaving it on?

Reply:
What makes you think this is an _internal_ CLI?

I've got this plugin running on a local environment, outside of VIP, and I can use the CLI commands just fine. We must remember that this plugin could be active on SiteGround, WPEngine, Pantheon, or other self-hosted hosts which allow SSH + WP-CLI access.

Reply:
@GaryJones Good point. Sorry for the confusion - I meant isolated the plugin itself. This only fires when the CSV is being loaded through our CLI function.

Reply:
I think you're right. I can't think of a case where the current script execution would be doing something else in parallel.

That said, I'd like to see a comment that explained that the change is only for the current request, and automatically restores to the default at the end of the execution.

Reply:
@rebeccahum I've added context regarding the scope of this change.